src/main/java/core/basesyntax/Helloworld.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+//import java.util.Objects; //Unused import
```

Comment:
```suggestion
```
if this import is unused, just delete it instead of commenting it

Reply:
Done! Thanks!